packages/list/index.tsx

```diff
@@ -217,15 +217,16 @@ export default class List extends React.Component<ListProps, ListState> {
       getListItemCount: () => this.listElements.length,
       getFocusedElementIndex: () =>
         this.listElements.indexOf(document.activeElement as HTMLLIElement),
+      getAttributeForElementIndex: (index, attr) => {
+        const listItem = this.listElements[index];
```

Comment:
does this also need an if (listItem) check like `setAttributeForElementIndex`?

Reply:
:ping_pong:

Reply:
Ah i forgot this ;)
I didn’t add check code because error message is very clear. ({index}th element is not exist)
If we catch that, I guess some of developers cannot know cause of error!